2-WebApp-graph-user/2-5-HybridFlow/wwwroot/js/site.js

```diff
@@ -0,0 +1,24 @@
+﻿const msalInstance = new msal.PublicClientApplication({
+    auth: {
+        clientId: '[YOUR CLIENT ID]',
+        redirectUri: 'https://localhost:7089/',
+        authority: 'https://login.microsoftonline.com/[YOUR DOMAIN OR TENANT ID HERE]',
+        postLogoutRedirectUri: 'https://localhost:7089/',
+    },
+    cache: {
+        cacheLocation: 'sessionStorage',
+        storeAuthStateInCookie: false,
+    }
+});
```

Comment:
Will likely update this.

There's a way to add in the configuration data through powershell scripting and I want to capture that here.